chapter-2.md

```diff
@@ -0,0 +1 @@
+Cold and naked was not what Dicky liked to be. But cold and naked was what Dicky was, and so he resigned himself to it. The rough and ridged sheet metal of the boxcar floor created and interesting and not unpleasant sensation when Dicky scooted across it to open the boxcar door. He looked forward to spending long hours searching the web for tetanus symptoms and applying them to himself later. He would hopefully be clothed by then. And maybe warm too, in a cozy internet cafe with a steel-haired waitress named Loretta who took neither flack nor guff. But in order to get to then, he had to leave the now. And leaving the now meant jumping out of a moving train. Oh well, he thought. Here goes Dicky!
```

Comment:
`The rough and ridged sheet metal of the boxcar floor created and interesting and not unpleasant sensation when Dicky scooted across it to open the boxcar door`

`created and interesting`

`and interesting`

`and`